tests/helpers/Participant.ts

```diff
@@ -0,0 +1,290 @@
+/* global APP $ */
+
+import { multiremotebrowser } from '@wdio/globals';
+
+import { IConfig } from '../../react/features/base/config/configType';
+import { urlObjectToString } from '../../react/features/base/util/uri';
+import Filmstrip from '../pageobjects/Filmstrip';
+import Toolbar from '../pageobjects/Toolbar';
+
+import { LOG_PREFIX, logInfo } from './browserLogger';
+import { IContext } from './participants';
+
+/**
+ * Participant.
+ */
+export class Participant {
+    /**
+     * The current context.
+     *
+     * @private
+     */
+    private context: { roomName: string; };
+    private _name: string;
+    private _endpointId: string;
+
+    /**
+     * The default config to use when joining.
+     *
+     * @private
+     */
+    private config = {
+        analytics: {
+            disabled: true
+        },
+        debug: true,
+        requireDisplayName: false,
+        testing: {
+            testMode: true
+        },
+        disableAEC: true,
```

Comment:
Weird that AGC is also not disabled, is that intentional?

disableAP disables them all